packages/react/src/lib/positioner/UpdatableComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import { Ref } from '@stardust-ui/react-component-ref'
+
+import { Extendable } from '../../types'
+
+interface UpdatableListProps {
+  /**
+   * Component that will be rendered.
+   */
+  Component: React.ComponentType
+
+  /**
+   * Called when a child component will be mounted or updated.
+   *
+   * @param {HTMLElement} node - Referred node.
+   */
+  innerRef?: React.Ref<HTMLElement>
+
+  /**
+   * Function that will trigger the rerender.
+   */
+  scheduleUpdate: Function
+
+  /**
+   * Array of conditions to be met in order to trigger a subsequent render.
+   */
+  updateDependencies: any[]
+}
+
+const UpdatableComponent: React.FunctionComponent<Extendable<UpdatableListProps>> = props => {
+  const { Component, innerRef, scheduleUpdate, updateDependencies, ...rest } = props
+
+  React.useEffect(() => scheduleUpdate && scheduleUpdate(), updateDependencies)
```

Comment:
```suggestion
  React.useEffect(() => scheduleUpdate(), updateDependencies)
```

It defined as required in `UpdatableListProps`...

Reply:
good point, applied suggestion, thanks